tests/test_resources/cwp-produced-images/config/packager-config-jdk11-more-specific.yml

```diff
@@ -0,0 +1,84 @@
+bundle:
+  groupId: "io.jenkins.tools.custom-war-packager.demo"
+  artifactId: "jenkinsfile-runner"
+  vendor: "Jenkins project"
+  title: "Jenkinsfile Runner demo"
+  description: "Jenkinsfile Runner Docker Image, produced by Custom WAR Packager"
+buildSettings:
+  jenkinsfileRunner:
+    source:
+      groupId: "io.jenkins"
+      artifactId: "jenkinsfile-runner"
+      build:
+        noCache: true
+      source:
+        dir: ./.jenkinsfile-runner-test-framework/source
+    docker:
+      base: "jenkins/jenkins:2.164-jdk11"
+      tag: "jenkins-experimental/jenkinsfile-runner-demo"
+      build: false
+war:
+  groupId: "org.jenkins-ci.main"
+  artifactId: "jenkins-war"
+  source:
+    version: "2.164"
```

Comment:
Why not 2.164.1? I think it's the first LTS java-11 compatible.

Reply:
It's the same one that in the previous jdk 11 configuration we were using.

Reply:
But 2.164.1 wasn't released at that moment. Maybe is it worth updating?